### PR TITLE
fix: ctx.var.cookie_* cookie not found log

### PR DESCRIPTION
**Pull request by slene**
### What this PR does / why we need it:

fix: #4516

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by slene** [comment]
@spacewander review plz

**Review by Yiyiyimu — APPROVED** [review]

File: t/core/ctx.t [hunk 1]
```diff
@@ -83,8 +83,8 @@ arg_a: aaa
 GET /t?a=aaa
 --- response_body
 cookie_host: nil
---- error_log
-failed to fetch cookie value by key: cookie_host error: no cookie found in the current request
+--- no_error_log
+[error]
```
Comment (spacewander) [inline t/core/ctx.t:87]:
```suggestion
failed to fetch cookie value by key
```

Comment (slene) [reply t/core/ctx.t:87]: ❓

Comment (spacewander) [reply t/core/ctx.t:87]: Change `[error]` to `failed to fetch cookie value by key`

Comment (slene) [reply t/core/ctx.t:87]: Why ? Ci is passed and no error.

Comment (spacewander) [reply t/core/ctx.t:87]: **You don't actually check it.** The original log message is a warning.

**Review by spacewander — APPROVED** [review]

**Pull request merged**